### PR TITLE
BUG: fix globbed csv to chunked dataframe

### PR DESCRIPTION
**Pull request by llllllllll**
Fixes the test failure in blaze. This adds a test in odo to check for this.

### CODE REVIEW

File: odo/backends/csv.py [hunk 1]
```diff
@@ -366,8 +367,19 @@ def resource_glob(uri, **kwargs):
 @convert.register(chunks(pd.DataFrame), (chunks(CSV), chunks(Temp(CSV))),
                   cost=10.0)
 def convert_glob_of_csvs_to_chunks_of_dataframes(csvs, **kwargs):
```
Comment (mtydykov) [inline odo/backends/csv.py]: Does this need a docstring?

File: odo/backends/csv.py [hunk 2]
```diff
@@ -366,8 +367,19 @@ def resource_glob(uri, **kwargs):
...
-    data = [partial(convert, chunks(pd.DataFrame), csv, **kwargs) for csv in csvs]
-    return chunks(pd.DataFrame)(data)
+    f = partial(convert, chunks(pd.DataFrame), **kwargs)
+
+    def df_gen():
+        dsk = {}
+        p = []
+        for n, csv_ in enumerate(csvs):
```
Comment (mtydykov) [inline odo/backends/csv.py]: It seems like we're just building a dictionary that tells `dsk_get` to apply the same function to all csv objects. Why is that necessary?

File: odo/backends/csv.py (hunk 1 above)
Comment (llllllllll) [inline odo/backends/csv.py]: No, this is an internal function so it is fine. Also, the signature is implied because it is a `convert` dispatch

File: odo/backends/csv.py (hunk 2 above)
Comment (llllllllll) [inline odo/backends/csv.py]: `dask.threaded.get` will run these function calls concurrently.

Comment (mtydykov) [inline odo/backends/csv.py]: Ok. Then it might be good to add a comment that explains why the loop is necessary.
